CMakeLists.txt

```
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.3)
+cmake_minimum_required (VERSION 3.10)
 project (hpGEM C CXX Fortran)
```

I think we are not using Fortran anymore now that our inline blas and lapack have been removed.

```suggestion
project (hpGEM C CXX)
```